client/my-sites/invites/p2/invite-accept-logged-out.jsx

```diff
@@ -11,7 +11,7 @@ function renderFooterLink( props ) {
 			<div>{ props.translate( 'Already have a WordPress.com account?' ) }</div>
 			<LoggedOutFormLinks>
 				<LoggedOutFormLinkItem onClick={ props.onClickSignInLink }>
-					{ props.translate( 'Log in instead.' ) }
+					{ props.translate( 'Log in instead' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.